spec/features/reply_from_councillor_spec.rb

```diff
@@ -14,9 +15,8 @@
     end
   end
 
-  background { comment.confirm! }
-
   scenario "it's entered by an admin and displayed on the application page" do
+    comment.confirm!
```

Comment:
@davwood this looks fine, but could you explain why this change was necessary? That would be a great thing for the commit message ;-)